src/tools/periph/mkfatimage16.c

```diff
@@ -467,7 +467,7 @@ int main(int argc, char *argv[])
   bpb->num_heads = heads;
 
   if (!bootsect_file) {       // Assume writing for Dosemu / FreeDOS so v4 BPB
-    memcpy(buffer + 0x03, "DOSEMU10", 8);
+    memcpy(buffer + 0x03, "IBM  3.3", 8);
```

Comment:
But it may still help in some of those horrible
heuristical cases that you've introduced in another
patch.